source/deployment/ldap-group-sync.rst

```diff
@@ -0,0 +1,144 @@
+
+.. image:: ../images/Group_Members.png
+
+Users will be removed from the group on subsequent synchronizations. However, they will not be removed from the teams and channels that they were defaulted to in the synchronization. If a user is removed from a AD/LDAP group and then later re-added, they will be defaulted again into the teams and channels configured in the group. If a member is removed altogether from the AD/LDAP system, all their team and channel memberships will be removed. 
+
+..note:: 
+When a member removes themselves manually from a channel, that action is tracked in the Channel Member History table.  Users are not re-added to channels from which they previously removed themselves. 
+
+Managing groups
+---------------
+Once a group has been configured, it can be edited to change the default team and channels by clicking “Edit” on the group on the list view.  
+
+Deleting groups
+---------------
+Groups can be deleted by adjusting your AD/LDAP group filter to remove the group or by unlinking the group on the groups listing page. If you add the group back by re-adjusting the AD/LDAP group filter and link the group again on the group configuration page, the previous configurations will be available.
+
+Frequently Asked Questions
+----------------------------
+
+Why is AD/LDAP Group Sync in alpha?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We want to make sure we have tested this feature in environments that have different AD/LDAP system and group structures to ensure the feature works well in the different varieties of environments our clients have. 
+
+When will AD/LDAP Group Sync be in beta?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We expect AD/LDAP Group Sync to be in beta early in 2019. 
+
+LDAP group sync will be in beta for a period of time until: 
+1. Searching and support for managing large volumes of groups and users is released. 
+2. This feature has been tested on a system with 10,000s of concurrent active users.
+
+
+Why can't my users see the teams and channels they have been synced to?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Users are synced to channels on their first login and on scheduled synchronization jobs as defined in the AD/LDAP configuration.  If the user is already a user within the system prior to the configuration of the teams and channels for the group, the next scheduled synchronization job will add the teams and channels.  
+
+You can manually initiate a synchronization from **System Console > Authentication > AD/LDAP > AD/LDAP Synchronize Now**.  Users may need to refresh to see the updates. 
+
+
+How do nested groups work with AD/LDAP Group Sync?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Users within nested groups are included as members of parent groups. The group filter that you specify can include any type of LDAP group on your system. The ``memberof`` LDAP attribute is used to determine nested groups that belong to a parent group.
```

Comment:
No we don't use `memberOf` at all. We use the `member` attribute. It's an important distinction.

Reply:
Updated.  Thanks.